samples/shutdown_vm.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+"""
+vSphere Python SDK program for shutting down VMs
+"""
+from __future__ import print_function
+
+from pyVim.connect import SmartConnect, Disconnect
+from pyVmomi import vim, vmodl
+
+import argparse
+import atexit
+import getpass
+import sys
+import ssl
+
+
+def GetArgs():
+    """
+    Supports the command-line arguments listed below.
+    """
+
+    parser = argparse.ArgumentParser(
+        description='Process args for shutting down a Virtual Machine')
+    parser.add_argument('-s', '--host', required=True,
+                        action='store', help='Remote host to connect to')
+    parser.add_argument('-o', '--port', type=int, default=443,
+                        action='store', help='Port to connect on')
+    parser.add_argument('-u', '--user', required=True, action='store',
+                        help='User name to use when connecting to host')
+    parser.add_argument('-p', '--password', required=False,
+                        action='store', help='Password to connect to host')
+    parser.add_argument('-v', '--vmname', required=True, action='append',
+                        help='Names of the Virtual Machines to shutdown')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    """
+   Simple command-line program for shutting down virtual machines on a system.
+   """
+
+    args = GetArgs()
+    if args.password:
+        password = args.password
+    else:
+        password = getpass.getpass(
+            prompt='Password to host %s and user %s:' % (args.host, args.user))
+
+    service_instance = None
+    context = ssl._create_unverified_context()
+    context.verify_mode = ssl.CERT_NONE
```

Comment:
I think this line can be removed. Also since _create_unverified_context only available since 2.7.9, it's better to do following:

context = None
if hasattr(ssl, '_create_unverified_context'):
    context = ssl._create_unverified_context()